pip/pep425tags.py

```diff
@@ -113,6 +113,9 @@ def get_supported(versions=None, noarch=False):
             for arch in arches:
                 supported.append(('%s%s' % (impl, versions[0]), abi, arch))
 
+        # Has binaries, does not use the Python API:
+        supported.append(('py%s' % (versions[0][0]), 'none', arch))
```

Comment:
I've noticed that this might go in the 'for arch in arches' loop that helps OSX folks out